src/Tribe/Aggregator/Admin_Bar.php

```diff
@@ -80,7 +80,13 @@ public function init( WP_Admin_Bar $wp_admin_bar = null ) {
 			return;
 		}
 
-		$service_response = Tribe__Events__Aggregator__Service::instance()->get_origins();
+		$service_response = get_transient( Tribe__Events__Aggregator::SERVICES_CACHE_KEY );
+
+		// Save HTTP request into a transient
+		if ( false === $service_response ) {
+			$service_response = Tribe__Events__Aggregator__Service::instance()->get_origins();
+			set_transient( Tribe__Events__Aggregator::SERVICES_CACHE_KEY, $service_response, DAY_IN_SECONDS );
```

Comment:
I have a feeling like we have some cache in place for the origins, or we had it in the past and was removed for reason. Or at least we need to invalidate cache once the user adds a new license.

Reply:
👍

Reply:
Based on the history of the file there seems that never has been stored in some sort of caching mechanism  (the response.)

Reply:
Pefecto!